ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -970,6 +968,9 @@ def _on_span_finished(span: Span, streamed_chunks):
         else:
             # best effort to join chunks together
             content = "".join([str(chunk) for chunk in streamed_chunks])
+        integration.llmobs_set_tags(span, args=args, kwargs=kwargs, response=content, operation="chain")
+        if span.error or not integration.is_pc_sampled_span(span):
```

Comment:
Another clarification question here about this - do we move the check for `span.error` after setting tags because the span may contain an error that happened mid-stream but we still want to capture the outputs that were returned?

Reply:
we re-check `span.error` in `llmobs_set_tags` for `llm`, `chat`, and `chain`, which i kept before adding output messages/value properties